### ISSUE TITLE
Missing checksums file for vale 2.0.0

### ISSUE THREAD

**Issue by themr0c** [vale-cli/vale#177]
```
/ # curl -sfLo vale.sh https://install.goreleaser.com/github.com/ValeLint/vale.sh
/ # bash vale.sh -b /usr/bin -d
ValeLint/vale info checking GitHub for latest tag
ValeLint/vale debug http_download https://github.com/ValeLint/vale/releases/latest
ValeLint/vale info found version: 2.0.0 for v2.0.0/Linux/64-bit
ValeLint/vale debug downloading files into /tmp/tmp.fOblnE
ValeLint/vale debug http_download https://github.com/ValeLint/vale/releases/download/v2.0.0/vale_2.0.0_Linux_64-bit.tar.gz
ValeLint/vale debug http_download https://github.com/ValeLint/vale/releases/download/v2.0.0/vale_2.0.0_checksums.txt
ValeLint/vale debug http_download_curl received HTTP status 404
```

**Comment by themr0c** [comment]
And v2.0.0-beta.3 does not work either:

```/ # bash vale.sh -b /usr/bin -d v2.0.0-beta.3
ValeLint/vale info checking GitHub for tag 'v2.0.0-beta.3'
ValeLint/vale debug http_download https://github.com/ValeLint/vale/releases/v2.0.0-beta.3
ValeLint/vale info found version: 2.0.0-beta.3 for v2.0.0-beta.3/Linux/64-bit
ValeLint/vale debug downloading files into /tmp/tmp.JDjFDg
ValeLint/vale debug http_download https://github.com/ValeLint/vale/releases/download/v2.0.0-beta.3/vale_2.0.0-beta.3_Linux_64-bit.tar.gz
ValeLint/vale debug http_download_curl received HTTP status 404
```
The released file has an additional `v`: https://github.com/ValeLint/vale/releases/download/v2.0.0-beta.3/vale_v2.0.0-beta.3_Linux_64-bit.tar.gz

**Comment by jdkato** [comment]
Thanks for the report.

This has been fixed (see #174).

**Comment by themr0c** [comment]
Awesome! Thank you!
